### PR TITLE
progress implementing database service

### PR DESCRIPTION
**Pull request by icahoon**

### CODE REVIEW

File: docs/design/database_service/cli.md [hunk 1]
```diff
@@ -40,19 +40,19 @@ where "database_name" is the name of the database instance.
 
 The commands available to a database instance are
 
-- **create**: Creates the database instance. If the database already exists this command will do nothing.
+- **create**:  Creates the database instance. If the database already exists this command will do nothing.
 - **destroy**: Destroyes the database instance. If the database does not exist the command will do nothing.
```
Comment (danielcahoon) [inline docs/design/database_service/cli.md]: Destroys

**Review by kwelch426 — APPROVED** [review]
Not part of this commit but Line 61 in docs/design/database_service/config.md..most should be changed to host.

File: pkg/arc/database_service.go [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+	"github.com/cisco/arc/pkg/provider"
+	"github.com/cisco/arc/pkg/resource"
+	"github.com/cisco/arc/pkg/route"
+)
+
+type databaseService struct {
+	*config.DatabaseService
+	arc             resource.Arc
+	dc              resource.DataCenter
+	databaseService resource.ProviderDatabaseService
```
Comment (danielcahoon) [inline pkg/arc/database_service.go]: Should this be providerDatabaseService?

**Comment by gstragand** [comment]
lgtm

**Review by danielcahoon — CHANGES_REQUESTED** [review]

**Review by kwelch426 — APPROVED** [review]

**Comment by icahoon** [comment]
Associated with Issue #53

**Pull request merged**